packages/typespec-go/test/http-specs/authentication/unionauthgroup/union_client_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package unionauthgroup_test
+
+import (
+	"context"
+	"testing"
+	"time"
+	"unionauthgroup"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+	"github.com/stretchr/testify/require"
+)
+
+type fakeCredential struct{}
+
+func (mc fakeCredential) GetToken(ctx context.Context, options policy.TokenRequestOptions) (azcore.AccessToken, error) {
```

Comment:
Just a syntax FYI: you don't have to name parameters of a function that doesn't use them
```suggestion
func (fakeCredential) GetToken(context.Context, policy.TokenRequestOptions) (azcore.AccessToken, error) {
```
(this is more obviously useful with function types)

Reply:
Thanks!! Fixed in [e44cf66](https://github.com/Azure/autorest.go/pull/1609/commits/e44cf66ce22c94dfd4ac62212951cd1c722ed0d7)